src/deployment-guides/tko-on-vsphere.md

```diff
@@ -645,6 +645,8 @@ To get the status of TKG Management cluster execute the following command:
     ![](img/tko-on-vsphere/image51.png)
 
 The TKG management cluster is successfully deployed and now you can created Shared Service and workload clusters  
+	
+> **_NOTE:_**  In version 1.4.x, you can receive a warning prompt for AKODeploymentConfig while deploying the management cluster. You can ignore the warning, for now, it will not create any issue.
```

Comment:
We should remove this as this deployment guide is targeting 1.5.x, and 1.4.x has already been shipped.

Reply:
This is already fixed in TKO 1.5 and as Craig said, we will not modify TKO 1.4 doc.